packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -1,22 +1,32 @@
 import { isValidPhoneNumber } from "libphonenumber-js";
 import z from "zod";
 
+import type { ALL_VIEWS } from "@calcom/features/form-builder/FormBuilderFieldsSchema";
 import type { eventTypeBookingFields } from "@calcom/prisma/zod-utils";
 import { bookingResponses } from "@calcom/prisma/zod-utils";
 
 type EventType = Parameters<typeof preprocess>[0]["eventType"];
-export const getBookingResponsesPartialSchema = (eventType: EventType) => {
+// eslint-disable-next-line @typescript-eslint/ban-types
+type View = ALL_VIEWS | (string & {});
```

Comment:
Hack to allow ALL_VIEWS to be not merged inside string. View would actually be "ALL_VIEW" | string due to & {}